Update for Xcode 14.1+

sf-luke-cha opened the pull request:
### Background
Apple announced that starting April 25, 2023, iOS, iPadOS, and watchOS apps submitted to the App Store must be built with Xcode 14.1 or later(https://developer.apple.com/news/?id=jd9wcyov ).
It should be verified that all iOS SampleApps in Sendbird run on Xcode14.1+.

### Summary
- [Change the ios version(9.0 -> 11.0) of Podfile](https://github.com/sendbird/sendbird-uikit-ios/commit/b83ef915c1ca0c912ebc1611e0a1fa9ec06b8077)


### Further Comments

tezpark commented:
@sf-luke-cha In the PR title, iOS version needs to be changed to iOS 11, not iOS 12

sf-luke-cha commented:
> @sf-luke-cha In the PR title, iOS version needs to be changed to iOS 11, not iOS 12

I'm sorry. I modified it to iOS11, but I wrote the wrong commit message. I fixed it.

tezpark reviewed (approved):
LGTM 👍
